lib/ansible/modules/cloud/openstack/os_quota.py

```diff
@@ -310,7 +311,10 @@ def _get_compute_quotas(cloud, project):
 def _get_quotas(cloud, project):
 
     quota = {}
-    quota['volume'] = _get_volume_quotas(cloud, project)
+    try:
+        quota['volume'] = _get_volume_quotas(cloud, project)
+    except exceptions.EndpointNotFound:
+        pass
```

Comment:
This now prints
```
TASK [Set quotas] ************************************************************************************************************************************************************************************************************************************************************
 [WARNING]: No public endpoint for volumev2 service was found. Ignoring volume quotas.
```

Same for network quotas since there might be deployments using nova network rather than neutron.

compute quotas remain outside of a try clause because compute service is expected to be in every openstack deployment.